### ISSUE TITLE
bbox for generating tiles

### ISSUE THREAD

**Issue by NieneB** [pka/t-rex#15]
Hi! Great tool! 
We try to generate pbf tiles for only the Netherlands and just a few zoom levels. When using `t_rex generate ` it will always generate tiles for the whole world/gridset and has to start at zoom level 0.

Is it possible to include settings for a bounding box and zoom levels for which it will generate the tiles in?  Or would it be possible to add these variables? Like for example in tl copy from https://github.com/mojodna/tl 
`-z 6 -Z 8 -b  "-124.4096 32.5343 -114.1308 42.0095" ` 

Thanks!

**Comment by pka** [comment]
`t_rex generate` has an option `--extent=[minx,miny,maxx,maxy]` and zoom level limits. 
Type `t_rex generate --help` for all options.

**Comment by NieneB** [comment]
yes we found it already. Maybe add it to the documentation? So it is easier to find. 
Anyway it is not working correctly.. `t_rex generate` with zoom levels works fine. But when setting an `--extent` it generates the wrong tiles. Looks like the y-axis is flipped then. Could this be the same as is happening in issue #16 at zoom level 16?

**Comment by pka** [comment]
Can you try again with the latest version, which includes a fix for #16.

**Comment by NieneB** [comment]
I tried again with the latest version but the `--extent` option is still not working for me. 
This is what I run:

```
 t_rex generate --config config.cfg --minzoom=8  --maxzoom=10 --extent=3777.56168633951,6097265.91823491,1281855.12787992,7341398.32595843
```

I get this error: 
```
2017-03-20 12:00:36.071 INFO Reading configuration from 'config.cfg'
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: ParseFloatError { kind: Invalid }', /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/libcore/result.rs:837
note: Run with `RUST_BACKTRACE=1` for a backtrace.
```

Running `t_rex generate` with the `--minzoom --maxzoom` options does work. The problem starts when adding the `--extent`

**Comment by pka** [comment]
The `extent` option was severely broken. Fixed in 0.7.1
